### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by EvilOlaf**
Enhances convenience to investigate user reports by avoiding useless questions about needed information like plugin version or similar.
Provided hints about properly creating the issue report will disappear after the report has been created successfully.

Maybe you find it useful even though there are not that much issue reports for this plugin as it is quite reliable.

### CODE REVIEW

**Comment by confuser** [comment]
Perhaps add in an online/offline mode question as well? It seems to be quite common that people with bungeecord then incorrectly set BM into offline mode, which causes more problems.

**Comment by EvilOlaf** [comment]
This should be covered by the config file but yeah, it is indeed quite common.

**Comment by confuser** [comment]
It already is, but because of the way Bukkit's YAML library handles comments, it doesn't show.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ Server software (CB/Spigot/Sponge/Thermos...):
...
 BanManager config.yml: 
-<!--- Use pastebin.com or similar to provide your config file. Make sure to -->
-<!--- garble/remove any MySQL credentials before uploading your file! -->
+Are you running offline mode?: 
+If yes, are you running your server behind Bungeecoord with online mode?: 
+<!--- Hint: Use pastebin.com or similar to provide your config file. Make -->
+<!--- sure to garble/remove any MySQL credentials before uploading your file! -->
 
 
 **Descripe your issue:**
```
Comment (Kakifrucht) [inline .github/ISSUE_TEMPLATE.md]: Describe!

Comment (EvilOlaf) [inline .github/ISSUE_TEMPLATE.md]: mh

**Pull request merged**